app/services/push_notifications/event_notifier/post_logic.rb

```diff
@@ -0,0 +1,24 @@
+module PushNotifications
+  class EventNotifier
+    class PostLogic
```

Comment:
I would try to find a better name for this. `PostLogic` is rather vague and as a reader, doesn't tell me much of what to expect from this class.

Reply:
Yeah, I don't like the name either... but I wanted to move forward... let's think about a better name later.